packages/fuels-accounts/src/accounts_utils.rs

```diff
@@ -13,10 +18,11 @@ pub fn calculate_base_amount_with_fee(
     tb: &impl TransactionBuilder,
     consensus_params: &ConsensusParameters,
     previous_base_amount: u64,
-) -> u64 {
+) -> Result<u64> {
     let transaction_fee = tb
-        .fee_checked_from_tx(consensus_params)
-        .expect("Error calculating TransactionFee");
+        .fee_checked_from_tx(consensus_params)?
+        // TODO: use better error
```

Comment:
I think this is worth creating a separate issue and linking it here.

Reply:
I also forgot to delete this.